rules-tests/CodeQuality/Rector/Return_/SimplifyReturnExpressionToConstantTypeRector/Fixture/multiple_modified_returns.php.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Return_\SimplifyReturnExpressionToConstantTypeRector\Fixture;
+
+class MultipleModifiedReturns
+{
+    public function run()
+    {
+        $return = false;
+
+        if (rand(0,1)) {
+            $return = null;
+
+            if (rand(2,1)) {
+                return $return;
+            }
+
+            return true;
+        }
+
+        return $return;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Return_\SimplifyReturnExpressionToConstantTypeRector\Fixture;
+
+class MultipleModifiedReturns
+{
+    public function run()
+    {
+        $return = false;
```

Comment:
I intentionally don't clean up stray variables.. I think those will be cleaned up by a code-style tool